.swcrc

```
@@ -0,0 +1,30 @@
+{
+    "$schema": "http://json.schemastore.org/swcrc",
+    "test": ".*.ts$",
+    "exclude": [
+        ".*.js$",
+        ".*.map$"
+    ],
+    "module": {
+        "type": "commonjs",
+        "strict": true,
+        "strictMode": true,
+        "lazy": false,
+        "noInterop": false
+    },
+    "jsc": {
+        "parser":
```

can we move the parenthesis here so that is coherent with the rest of the file?

```suggestion
        "parser": {
```